haystack/utils/highlighting.py

```diff
@@ -157,9 +149,15 @@ def render_html(self, highlight_locations=None, start_offset=None, end_offset=No
         highlighted_chunk += text[matched_so_far:]
 
         if start_offset > 0:
-            highlighted_chunk = '...%s' % highlighted_chunk
+            if self.trim_text:
+                highlighted_chunk = '...%s' % highlighted_chunk
```

Comment:
Wondering whether this should be a Unicode ellipsis character (U+2026, `u'\N{HORIZONTAL ELLIPSIS}'`)